### PR TITLE
922442_ContextMenu_UG

### PR DESCRIPTION
**Pull request by Yaminisrisf4389**
Description - Add UG documentation for Customized context menu

### CODE REVIEW

**Review by Vikassekar — CHANGES_REQUESTED** [review]
As discuss in the call can you please modify the content ?

File: wpf/Pdf-Viewer/Customizing-context-menu.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+Please refer to the following example for more details.
+### Add the context menu item.
+You can enhance the existing default context menu by adding new menu items to it or inserting them at specific positions based on the source type that triggers the menu. This allows for dynamic customization to suit various use cases, such as adding custom options for annotations, redactions, text selection, or signature fields.
+{% tabs %}
+{% highlight c# %}
+  private void PdfViewer_ContextMenuOpening(object sender, ContextMenuOpeningEventArgs e)
+  {
+      if (e.Source == "Annotation")
+      {
+	    e.MenuItems.Insert(0,(new ContextMenuItem { Content = "Lock Annotation" }));
```
Comment (Vikassekar) [inline wpf/Pdf-Viewer/Customizing-context-menu.md:56]: image add and content are mismatched , can you change the screen shot for add

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: wpf/Pdf-Viewer/Customizing-context-menu.md (hunk 1 above)
Comment (Yaminisrisf4389) [reply wpf/Pdf-Viewer/Customizing-context-menu.md:56]: I changed the screen shot for adding menu item

**Review by Vikassekar — APPROVED** [review]
Approving this

**Pull request merged**